### ISSUE TITLE
Open TOS attachments and TOS reference link on a new page

### ISSUE THREAD

**Issue by aetdr** [globaleaks/GlobaLeaks#3097]
Hi

When a TOS is included in a questionnaire it is possible to provide attachments to the TOS by adding a URL.

If the link is path to a file (pdf for example), when clicked on, the file gets downloaded/opened in new window.
But if the link is an URL to a website than the whistleblower is sent to the site loosing all the typed information.

Is it possible to add `target="_blank"` to this link, and of course `ref="noopener noreferrer"`?

```
<div data-ng-switch-when="tos">
        <div class="form-group tos-text" data-markdown-to-html="field.attrs.text.value"></div>
        <div data-ng-if="field.attrs.attachment.value">
          <a href="{{field.attrs.attachment_url.value}}" download>{{field.attrs.attachment_text.value}}</a>
        </div>
```

**Comment by evilaliv3** [comment]
Thank you for the suggestion, i thin we could add absolutely target="_blank" to avoid to loose the typed information.

I'm in doubt if to add ref="noopener noreferrer" for these reasons:
1. noreferrer includes noopener so actually using noreferrer would be enough.
2. actually we are setting the referrer policy via the HTTP Header Referrer-Policy that is set to value no-referrer, that means any link will work with policy noreferrer (that includes policy noopener)

My main doubt is about 2. What do you think?

**Comment by evilaliv3** [comment]
Closing; This improvement will be released in release 4.5.2
